plugins/filter/jc.py

```diff
@@ -94,7 +94,12 @@ def jc(data, parser, quiet=True, raw=False):
         dictionary or list of dictionaries
 
     Example:
-
+        # This step is optional in case you have the package already installed on your machine
+        - name: add the jc prereqs to the running machine
+          delegate_to: 127.0.0.1
```

Comment:
Always use `localhost` and not `127.0.0.1` or `::1`.
```suggestion
          delegate_to: localhost
```